packages/harpguru-core/src/components/menu-open-button/menu-open-button.tsx

```diff
@@ -0,0 +1,55 @@
+import Animated, { add } from 'react-native-reanimated'
+import { TapGestureHandler } from 'react-native-gesture-handler'
+import { View, StyleSheet } from 'react-native'
+import React from 'react'
+
+import { getMenuStylesAndAnimationVals } from '../../utils'
+import type { MenuProps } from '../../types'
+
+import { useTapAnimation } from './hooks'
+
+type LocalMenuProps = MenuProps & {
+  readonly children: React.ReactNode
+}
+export const MenuOpenButton = ({
+  hideMenu,
+  hideLabel,
+  stashPosition,
+  openCloseTapHandler,
+  children,
+}: LocalMenuProps): React.ReactElement => {
+  const { labelCounterScale, labelProtrusion } = getMenuStylesAndAnimationVals(
+    hideMenu,
+    hideLabel,
+    stashPosition
+  )
+
+  const styles = StyleSheet.create({
```

Comment:
Same comment about `style` vs `styles` as we have above. It might be less applicable here because the style is on a particular element from a bunch. I don't like the name of `styles.label` anyway so something like `styles.icon` can be used at least.

Reply:
Done